pkg/cmd/server/kubernetes/node_config.go

```diff
@@ -187,9 +187,6 @@ func BuildKubernetesNodeConfig(options configapi.NodeConfig) (*NodeConfig, error
 	if value := cmdutil.Env("OPENSHIFT_DIND", ""); value == "true" {
 		glog.Warningf("Using FakeOOMAdjuster for docker-in-docker compatibility")
 		cfg.OOMAdjuster = oom.NewFakeOOMAdjuster()
-		glog.Warningf("Disabling cgroup manipulation of nested docker daemon for docker-in-docker compatibility")
-		cfg.RuntimeCgroups = ""
```

Comment:
Yes, field name changed, but default also changed to empty, so it wasn't
needed.



On Wed, Mar 16, 2016 at 8:03 AM, David Eads <notifications@github.com>
wrote:

> In pkg/cmd/server/kubernetes/node_config.go
> <https://github.com/openshift/origin/pull/7975#discussion_r56321794>:
>
> > @@ -187,9 +187,6 @@ func BuildKubernetesNodeConfig(options configapi.NodeConfig) (*NodeConfig, error
> >  	if value := cmdutil.Env("OPENSHIFT_DIND", ""); value == "true" {
> >  		glog.Warningf("Using FakeOOMAdjuster for docker-in-docker compatibility")
> >  		cfg.OOMAdjuster = oom.NewFakeOOMAdjuster()
> > -		glog.Warningf("Disabling cgroup manipulation of nested docker daemon for docker-in-docker compatibility")
> > -		cfg.RuntimeCgroups = ""
>
> This has been both added and removed in your rebase. Intentional?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/7975/files/6401e0e510ff9c48b250a7d23444f768dc7be254..a3ba63351a644d8d81c32a6f624cc37bd82e05ea#r56321794>
>